Makefile

```diff
@@ -37,5 +40,41 @@ endif
 release-test:
 	./test.sh
 
+.PHONY: sysroot-pack
+sysroot-pack:
+	@tar cf - $(SYSROOT_DIR) -P | pv -s $[$(du -sk $(SYSROOT_DIR) | awk '{print $1}') * 1024] | pbzip2 > $(SYSROOT_ARCHIVE)
+
+.PHONY: sysroot-unpack
+sysroot-unpack:
+	@pv $(SYSROOT_ARCHIVE) | pbzip2 -cd | tar -xf -
```

Comment:
I didn't see anywhere that these functions are explicitly used, they just came from https://github.com/goreleaser/goreleaser-cross-example/blob/master/Makefile#L7-L13.  These could probably be removed.